admin/services/group.service.js

```diff
@@ -1,11 +1,27 @@
 'use strict'
 
 const groupDataService = require('../services/data-access/group.data.service')
+const pupilDataService = require('../services/data-access/pupil.data.service')
 
 const groupService = {}
 
 groupService.getGroups = async function (query) {
   return groupDataService.fetchGroups(query)
 }
 
+groupService.getPupils = async function (schoolId) {
+  if (!schoolId) { return false }
+  return pupilDataService.getSortedPupils(schoolId)
+}
+
+groupService.getGroupById = async function (groupId) {
+  if (!groupId) { return false }
+  return groupDataService.fetchGroup({'_id': groupId})
```

Comment:
pupilDataService uses `getXXX` 
groupDataService uses `fetchXXX`
is that intentional?

Reply:
Yes, it's to not make it confusing (I hope?)

Reply:
i would go with the standard get, otherwise its inconsistent with other data services